index.d.ts

```diff
@@ -42,9 +42,12 @@ hexRgb('#cd2222cc');
 
 hexRgb('#cd2222cc', {format: 'array'});
 //=> [205, 34, 34, 0.8]
+
+hexRgb('#cd2222cc', {format: 'css'});
+//=> 'rgba(205, 34, 34, 0.8)'
```

Comment:
I think it should use the modern CSS format: https://twitter.com/alexjmedick/status/1254153232823263235